Lib/ctypes/__init__.py

```diff
@@ -205,6 +205,12 @@ class c_longdouble(_SimpleCData):
 if sizeof(c_longdouble) == sizeof(c_double):
     c_longdouble = c_double
 
+try:
+    class c_double_complex(_SimpleCData):
+        _type_ = "C"
```

Comment:
I looked a bit around the "fielddesc" but what I said before on the `tag*` structure was incorrect (I updated my comment). Having `Cf` here is actually a bit trickier because `fielddesc` needs to be changed and everything that is using it as well so I'm not sure whether we want to do it now (and I think it's best to do it in another PR).

Do you want me to work on that or do you think we should just, for now, use "C" and possibly change it when we decide to support float and long double complex?

Reply:
Yes, "fielddesc" uses char...

> Do you want me to work on that or do you think we should just, for now, use "C" and possibly change it when we decide to support float and long double complex?

Well, we are not out of chars.  The ``_type_`` is not a public property, anyway.

Reply:
Oh ok! Well then I think we don't need to bother with that then!